src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -1,11 +1,12 @@
 package core.basesyntax.bookstore.dto.book;
 
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title,
-                                   String author,
-                                   String isbn,
-                                   BigDecimal price,
+public record CreateBookRequestDto(@NotNull String title,
```

Comment:
for String use `@NotBlank`

```suggestion
public record CreateBookRequestDto(@NotBlank String title,
```